web-ui/src/helpers/strings.test.js

```diff
@@ -0,0 +1,10 @@
+import { titleCase } from './strings';
+
+describe('strings', () => {
+  it('can title-case a string', () => {
+    expect(titleCase('')).toBe('');
+    expect(titleCase('a')).toEqual('A');
+    expect(titleCase('test')).toBe('Test');
+    expect(titleCase('one two three')).toBe('One Two Three');
+  });
+});
```

Comment:
Typically for string formatting utils I'd use a name like `format.js` for the helper. This allows for the file to format more than just strings without needing to be renamed later, or a new file created for other types of formatting.